MODULE.bazel

```diff
@@ -10,12 +10,12 @@ bazel_dep(name = "bazel_features", version = "1.15.0")
 bazel_dep(name = "bazel_skylib", version = "1.3.0")
 bazel_dep(name = "platforms", version = "0.0.5")
 bazel_dep(name = "rules_python", version = "0.23.1")
+bazel_dep(name = "rules_cc", version = "0.0.9")
```

Comment:
`rules_cc` shouldn't be required at all - I'll put in a separate PR to rectify this

Reply:
Ah rules_cc as a dev_dependency is only present to satisfy the rbe configuration - that package needs some work to fix it up properly..

Reply:
@jsharpe without it I have the following error on Windows:
```
The repository '@@[unknown repo 'rules_cc' requested from @@rules_foreign_cc~~tools~glib_dev]' could not be resolved: No repository visible as '@rules_cc' from repository '@@rules_foreign_cc~~tools~glib_dev'. and referenced by '@@rules_foreign_cc~//toolchains/private:pkgconfig_tool_msvc_build_'
```
So it was added to fix the error above.

Reply:
I know but the solution isn't to start depending on `rules_cc`.  I'm addressing this in https://github.com/bazelbuild/rules_foreign_cc/pull/1269 instead.

Reply:
Okay, got it.

Reply:
I've merged 1269 now so if you remove this files changes from this PR then we can land this fix.

Reply:
@jsharpe I removed it.